### ISSUE TITLE
[A11Y] ALT parameter required for Issue cover image

### ISSUE THREAD

**Issue by israelcefrin** [pkp/pkp-lib#9417]
## Issue Description
When an editor uploads a cover image to an issue the "ALT" parameter is not shown. It causes that description field remains empty and it requires an extra step to edit the issue and add the ALT description content. This description is used by screen reader users to access the image content and it is usual that issue covers show photos or have an "Issue title" on the cover image.

## How to reproduce / Current behavior
// Step-by-step instructions explaining how to reproduce the problem. The environment required, explicit identification of the component in question
1. Log in to the journal
2. Click on "Issues"
3. Click on the "create issue" button
4. Roll down to the "Cover image" section
5. Upload an image
6. Click on save 
It doesn't show any alert of missing descriptive text.

## What application are you using?
 OJS version 3.4.0

## Stack used 
- Device: Laptop
- OS: MacOS
- Browser: Firefox
- Version: 118

## Testing method
- Method:  Manual - Keyboard

## Action
Adding the during the upload with validation to prevent it from being saved empty. The field must be  presented after uploading an image, for this reason, the validation must be attached to the upload event.

## Additional information
![image](https://github.com/pkp/pkp-lib/assets/4992902/f24121ed-cf80-435d-9837-a3abc38c40f7)

![image](https://github.com/pkp/pkp-lib/assets/4992902/dc5fa73c-9048-4c62-98fc-b7cfb4f846bd)

![image](https://github.com/pkp/pkp-lib/assets/4992902/05d9f8a7-0c50-46f5-9e5f-01e87342ec04)

**Comment by kaitlinnewson** [comment]
As noted in the discussion in https://github.com/pkp/immersion/pull/162, I wonder if it's a good idea to require alt-text instead of having a reasonable fallback (like we do in the default theme template), as some users are not aware of good practices for writing alt-text and it may result in the alt-text being less useful to a screenreader user.

**Comment by israelcefrin** [comment]
Hello @kaitlinnewson , I was reminded that our most recent audit report may affect deeply all this discussion. In OMP redundant links on book images and titles were flagged as a critical issue since it creates unnecessary extra work for screen reader users who have to navigate through duplicate link.
https://github.com/pkp/pkp-lib/issues/12665

Let's keep it open and most likely merging these issues as covered by the more recent one. We can use the same approach on every theme (either OJS or OMP) given that cover image for books or issues have the same concept.
